packages/shared/src/enterprise/license-consts.ts

```diff
@@ -70,6 +70,7 @@ export type SubscriptionInfo = {
   dateToBeCanceled: string;
   cancelationDate: string;
   pendingCancelation: boolean;
+  isVercelIntegration: boolean;
```

Comment:
I see Romain's PR has an `isVercelIntegraiton` on the org - I didn't know that when I made this - I can remove this and on the `/billing` page use the org.isVercelIntegration instead